great-idea/css/index.css

```diff
@@ -93,21 +92,30 @@ nav a {
 }
 
 .cta {
-	position: relative;
+	display: flex;
```

Comment:
Nice work here -- you seem to have a good understanding of CSS inheritance, and you're making use of the full range of flexbox properties.